code/modules/mob/living/living.dm

```diff
@@ -692,10 +694,8 @@
 						if(spec_color)
 							TH.color = spec_color
 
-/mob/living/carbon/human/makeTrail(turf/T, turf/start, direction, spec_color)
-	if((NOBLOOD in dna.species.species_traits) || !bleed_rate || bleedsuppress)
-		return
-	spec_color = dna.species.blood_color
+/mob/living/simple_animal/pet/dog/corgi/borgi/makeTrail(turf/T, turf/start, direction, spec_color)
+	spec_color = blood_color
```

Comment:
put this in `monkestation/code/modules/mob/living/simple_animal/friendly/dog.dm`